CONTRIBUTING.md

```diff
@@ -78,6 +78,40 @@ plugins {
 
 There is an automation script for this routine, see [testDokka.sh.md](scripts/testDokka.sh.md) for details.
 
+### Updating Gradle
```

Comment:
Okay, that doesn't look nice :)

Also, I've followed your [link](https://docs.gradle.org/current/userguide/gradle_wrapper.html#configuring_checksum_verification) and there is a CLI property `--gradle-distribution-sha256-sum` which could simplify things.

So I've tried to use it, and it looks like everything is not that complicated.
We can safely remove `distributionSha256Sum` from `tasks.wrapper` configuration (leaving only rewriting of URL), and so we will have two options on how to update Gradle:
1. run `gradle wrapper --gradle-version=8.8 --gradle-distribution-sha256-sum=a4b4158601f8636cdeeab09bd76afb640030bb5b144aafe261a5e8af027dc612` (twice, or just `gradle wrapper` second time)
2. manually update both version and checksum in `gradle-wrapper.properties` and just `gradle wrapper` (to update scripts)

Both options work fine and looks like no complicated setup is needed at all.
In the end, looks like we need to just drop specifying `distributionSha256Sum` in `tasks.wrapper`.

Reply:
That makes much more sense! Thanks. I'll update the instructions.